libdevcore/RLP.h

```diff
@@ -0,0 +1,487 @@
+/*
```

Comment:
Looks like the whole of RLP is dragged in purely for initializing EmptyTrie constant!

What constant should I replace this with?

h256 const EmptyTrie = sha3(rlp(""));